runtime/jvmti/jvmtiStackFrame.c

```diff
@@ -397,14 +397,15 @@ jvmtiGetFrameLocation(jvmtiEnv* env,
 	J9VMThread * currentThread;
 	jmethodID rv_method = NULL;
 	jlocation rv_location = 0;
+	J9InternalVMFunctions* vmFuncs = vm->internalVMFunctions;
 
 	Trc_JVMTI_jvmtiGetFrameLocation_Entry(env);
 
 	rc = getCurrentVMThread(vm, &currentThread);
 	if (rc == JVMTI_ERROR_NONE) {
 		J9VMThread *targetThread = NULL;
 
```

Comment:
`vmFuncs` can be defined here since it is only used inside this `if` statement.
```suggestion
		J9VMThread *targetThread = NULL;
		J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
```

Reply:
https://github.com/eclipse-openj9/openj9/pull/15753#discussion_r960671178 According to this Keith wants variables to be defined at the beginning of the block.

Reply:
`beginning of a block` refers to the start of a scope. otherwise, compilers on Windows throw an error.
```
if (rc == JVMTI_ERROR_NONE) {
  /* beginning of a block */  
```

Ref coding standard: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#variable-declarations